src/Cake.Common/Tools/WiX/WiXAliases.cs

```diff
@@ -9,7 +9,14 @@
 namespace Cake.Common.Tools.WiX
 {
     /// <summary>
-    /// Contains functionality related to running WiX tools.
+    /// <para>Contains functionality related to <see href="http://wixtoolset.org/">WiX</see>.</para>
+    /// <para>
+    /// In order to use the commands for this alias, include the following in your build.cake file to download and
+    /// install from NuGet.org, or specify the ToolPath within the appropriate settings class:
+    /// <code>
+    /// #tool "nuget:?package=WiX"
```

Comment:
There is no official release of this, but the most popular seems to be [WiX.Toolset](https://www.nuget.org/packages/WiX.Toolset). The official download page is http://wixtoolset.org/releases/